ui/view/controls/VersionFooter.qml

```diff
@@ -8,5 +8,14 @@ SFText {
     Layout.bottomMargin: 27
     font.pixelSize:      12
     color:               Qt.rgba(1, 1, 1, 0.3)
-    text:                [qsTrId("settings-version"), BeamGlobals.version()].join(' ')
-}
\ No newline at end of file
+    text:                qsTrId("settings-version") + BeamGlobals.version()
+
+    MouseArea {
+    anchors.fill: parent
+    acceptedButtons: Qt.LeftButton
+    cursorShape: Qt.PointingHandCursor
+    onClicked: function() {
+        BeamGlobals.copyToClipboard(qsTrId("settings-version") + BeamGlobals.version())
```

Comment:
The same: avoid duplication of `qsTrId("settings-version") + BeamGlobals.version()`